From 0cf3f92a09b725696a5a40c64ba8509e62764f85 Mon Sep 17 00:00:00 2001
From: Vincent Abriou <vincent.abriou@st.com>
Date: Fri, 17 Jun 2016 10:48:07 +0200
Subject: [PATCH 4/6] mali: replace get_unused_fd() by get_unused_fd_flags(0)

This patch replaces call to get_unused_fd() with equivalent call to
get_unused_fd_flags(0) to preserve current behavior for existing code.
Necessary for Android build.

Signed-off-by: Christophe Guibout <christophe.guibout@st.com>
---
 src/devicedrv/mali/linux/mali_sync.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/devicedrv/mali/linux/mali_sync.c b/src/devicedrv/mali/linux/mali_sync.c
index 8bec3a4..a6ec4d2 100755
--- a/src/devicedrv/mali/linux/mali_sync.c
+++ b/src/devicedrv/mali/linux/mali_sync.c
@@ -298,7 +298,7 @@ s32 mali_sync_fence_fd_alloc(struct sync_fence *sync_fence)
 {
 	s32 fd = -1;
 
-	fd = get_unused_fd();
+	fd = get_unused_fd_flags(0);
 	if (fd < 0) {
 		sync_fence_put(sync_fence);
 		return -1;
-- 
2.7.4

